Doc/library/stdtypes.rst

```diff
@@ -2058,7 +2058,7 @@ expression support in the :mod:`re` module).
 .. index::
    single: universal newlines; str.splitlines method
 
-.. method:: str.splitlines([keepends])
+.. method:: str.splitlines(keepends=False)
```

Comment:
Hmm, I'm not too sure about this. Maybe @rhettinger can comment please?

Reply:
The Argument Clinic definition is `splitlines($self, /, keepends=False)`
The proposed change is ok.

Reply:
Alright, thanks for finding out Andrew!